build.sh

```diff
@@ -0,0 +1,51 @@
+name="oval-graph"
```

Comment:
It is a good idea to have `set -e` at the beginning, so when there is an error and any command fails, the script aborts.